lib/base/novacom.js

```diff
@@ -334,48 +334,6 @@ const fs = require('fs'),
             }
         },
 
-        getSshPrvKey: function(target, next) {
```

Comment:
Did you delete this code because this code is not used no longer?

Reply:
Sure.